dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ClientRegressionIT.java

```diff
@@ -43,68 +40,70 @@
 import org.junit.experimental.categories.Category;
 import org.junit.rules.TemporaryFolder;
 
+import static io.dockstore.common.CommonTestUtilities.checkToolList;
 import static io.dockstore.common.CommonTestUtilities.runOldDockstoreClient;
+import static io.dockstore.common.CommonTestUtilities.version;
 
 /**
- * @author dyuen
+ *
```

Comment:
Same here, don't delete the comment. Replace and add to it, what does this class do?